### PR TITLE
Fix #37 Moved shelljs out of the global scope

### LINKED ISSUE #37: Remove deprecated usage of shelljs/global
According to the docs: https://github.com/shelljs/shelljs#global-vs-local, the usage of global is deprecated.

### PR DESCRIPTION
**Pull request by CaptainAchilles**

### CODE REVIEW

File: build/utils.js [hunk 1]
```diff
@@ -69,20 +69,21 @@ exports.styleLoaders = function (options) {
 };
 
 exports.apiSetup = function (options) {
-	var spinner = ora("setting up api...");
+	var spinner = ora("Setting up api...");
```
Comment (MrCeleryman) [inline build/utils.js:72]: Good catch!

File: build/utils.js [hunk 2]
```diff
@@ -69,20 +69,21 @@ exports.styleLoaders = function (options) {
...
-			console.error(`\nGO ${error}`);
-			return;
-		}
-	});
-	spinner.stop();
+	var goOutput = shell.exec("cd api && go get");
+	
+	console.log(goOutput.stdout);
+	if (goOutput.stderr) {
+		spinner.stop();
```
Comment (MrCeleryman) [inline build/utils.js:84]: Excellent changes here Nich 🎉

**Review by MrCeleryman — CHANGES_REQUESTED** [review]
Is shelljs also called in build.js might need to change it there too?

**Comment by CaptainAchilles** [comment]
Totally didn't see the use of shelljs in the build.js file. I went through and added a check to only copy assets from the static directory if it exists, and cleaned up the global namespace by assigning shelljs to the `shell` var.

**Review by MrCeleryman — APPROVED** [review]

**Comment by MrCeleryman** [comment]
Great work Nich! So glad you're on board as a collaborator 🥇

**Pull request merged**